5. [[Promise.catch()]] - The `catch()` method returns a Promise and deals with rejected cases only. It behaves the same as calling `Promise.then(undefined, onRejected)`.